src/Repository/RepositoryInterface.php

```diff
@@ -14,6 +14,15 @@
     public function find($id);
 
     /**
+     * Find multiple objects by identifier
+     *
+     * @param array $ids
+     *
+     * @return array
+     */
+    public function findMultiple(array $ids);
```

Comment:
Maybe `findById` would be better? Since `findBy` is assumed to return multiple already?